static/app/views/detectors/components/connectedAutomationList.tsx

```diff
@@ -3,25 +3,41 @@ import {TimeAgoCell} from 'sentry/components/workflowEngine/gridCell/timeAgoCell
 import {TitleCell} from 'sentry/components/workflowEngine/gridCell/titleCell';
 import {defineColumns, SimpleTable} from 'sentry/components/workflowEngine/simpleTable';
 import {t} from 'sentry/locale';
-import type {Automation} from 'sentry/views/automations/components/automationListRow';
+import type {Automation} from 'sentry/types/workflowEngine/automations';
+import {useAutomationActions} from 'sentry/views/automations/hooks/utils';
+import {AUTOMATIONS_BASE_URL} from 'sentry/views/automations/routes';
 
 const columns = defineColumns<Automation>({
   name: {
     Header: () => t('Name'),
-    Cell: ({value, row}) => <TitleCell name={value} link={row.link} />,
+    Cell: ({value, row}) => (
+      <TitleCell
+        name={value}
+        link={`${AUTOMATIONS_BASE_URL}/${row.id}/`}
+        projectId={row.detectorIds[0]}
+      />
+    ),
     width: 'minmax(0, 3fr)',
   },
   lastTriggered: {
     Header: () => t('Last Triggered'),
     Cell: ({value}) => <TimeAgoCell date={value} />,
   },
-  actions: {
+  actionFilters: {
     Header: () => t('Actions'),
-    Cell: ({value}) => <ActionCell actions={value} />,
+    Cell: ({row}) => {
+      const actions = useAutomationActions(row);
```

Comment:
Not sure if we need to abstract this into a hook if this is the only place we're using it?

Reply:
i'm thinking we'll need this later for the automation details page. would you prefer that i abstract it later once we need it in other places?